setup.cfg

```diff
@@ -38,6 +38,7 @@ install_requires =
     sqlalchemy>=2.0
     sqlalchemy-utils>=0.32
     typing-extensions
+    urllib3>=2.6.0
```

Comment:
So my preference would be to avoid adding dependencies here that are not directly used in the code (or as a work around for pip-tools not including platform specific requirements.) It creates the situation where it's hard to spot why a requirement is in the file if it's not used in the code. For this type of situation where we are just upgrading a downstream dependency I vote we just recompile the dependencies with
```shell
pip-compile \
    --index-url=https://nexus.osdc.io/repository/pypi-gdc-releases/simple \
    --strip-extras \
    -P urllib3;
```